notes/vim/copypaste.md

```diff
@@ -1,4 +1,15 @@
-## VIM Copy and Paste tricks
+## Copy and Paste
 
-Tricks here
+y=yank; d=delete; c=cut (which is "delete then insert mode")
 
+These are used for the start of a sequence, followed by what to yank, delete or cut.
+
+Examples:
+
+> yiw : yank inner word
+> dta : delete until the letter 'a'
+
+Double letters (yy, dd, etc) mean: do it for the entire line
+
+p=paste
+P=Paste above
```

Comment:
this seems to conflict with the copy paste note added in #28, do you want to overwrite that note?

Reply:
no, we we're working in parallel and... I think you're is slightly (!) better :)